hub/templates/hub/area/_place_data.html

```diff
@@ -126,6 +126,12 @@ <h5>{{ category.name }}</h5>
                                     {% endfor %}
                                 </tbody>
                             </table>
+                {% elif category.name == 'Wildlife Trusts Reserves' %}
+                    <ul class="tag-cloud">
+                        {% for data in category.data.json %}
+                        <li class="tag">{{ data.title }}</li>
+                        {% endfor %}
+                    </ul>
```

Comment:
@alexander-griffen ideally this would "just work" using the exact same template logic that currently displays "tag-cloud-like" datasets like the WI groups: https://github.com/mysociety/local-intelligence-hub/blob/baf243138f7aae5e90be7a2f1e80cc7446c787eb/hub/templates/hub/area.html#L384-L402